bugsnag-android-core/src/androidTest/java/com/bugsnag/android/ObserverInterfaceTest.java

```diff
@@ -129,15 +129,6 @@ public void testAddNullMetadataToMetaDataSendsMessage() {
         assertEquals("bar", metadataItem.get(1));
     }
 
-    @Test
```

Comment:
These tests are no longer needed as immutable properties don't need to sync with the native layer.